sample/kotlin/src/realm/kotlin/com/datadog/android/sample/data/db/LogRealm.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sample.data.db
+
+import android.provider.BaseColumns
+import io.realm.RealmObject
+import io.realm.annotations.PrimaryKey
+import io.realm.annotations.RealmField
+import java.util.UUID
+
+open class LogRealm(
+    @RealmField(name = BaseColumns._ID)
```

Comment:
I used to work with Realm a while back and… I definitely don't miss this XD

Reply:
Yeah definitely not the most friendly framework